scripts/exglobal_archive.py

```diff
@@ -31,16 +31,16 @@ def main():
             'restart_interval_gdas', 'restart_interval_gfs',
             'AERO_ANL_RUN', 'AERO_FCST_RUN', 'DOIBP_WAV', 'DO_JEDIOCNVAR',
             'NMEM_ENS', 'DO_JEDIATMVAR', 'DO_VRFY_OCEANDA', 'FHMAX_FITS',
-            'IAUFHRS', 'DO_FIT2OBS']
+            'IAUFHRS', 'DO_FIT2OBS', 'NET']
 
     archive_dict = AttrDict()
     for key in keys:
-        archive_dict[key] = archive.task_config[key]
+        archive_dict[key] = archive.task_config.get(key, "Undefined")
 
     # Also import all COMIN* and COMOUT* directory and template variables
     for key in archive.task_config.keys():
         if key.startswith("COMIN_") or key.startswith("COMOUT_"):
-            archive_dict[key] = archive.task_config[key]
+            archive_dict[key] = archive.task_config.get[key]
```

Comment:
```suggestion
            archive_dict[key] = archive.task_config.get(key, 'Undefined')
```

Reply:
Curious, why `Undefined` instead of `None`.  `None` is a reserved keyword and should help with `if`, `try`, etc. if needed later.

Reply:
`None` would probably be a better choice.  Just a note that jinja does not have a `None` type, so `if`, etc, will need to look for a `"None"` string.

Thinking about it a little more, perhaps the better way to go is to actually never create a key for the missing variable and allow `wxflow` to handle the creation of `Undefined` objects.  This has to be a `try` block, though, as a missing key raises an exception.

```suggestion
            try:
                archive_dict[key] = archive.task_config.get(key)
```